app/src/main/java/com/github/factotum_sdp/factotum/ui/signup/SignupViewModel.kt

```diff
@@ -0,0 +1,7 @@
+package com.github.factotum_sdp.factotum.ui.signup
+
+import androidx.lifecycle.ViewModel
+
+class SignupViewModel : ViewModel() {
+    // TODO: Implement the ViewModel
```

Comment:
Is this something that had to be done before the merge ? Or a remainder for later ?

Reply:
It will be implemented in the next task